### PR TITLE
Add api for waiting web fonts (T736793)

### PR DESCRIPTION
**Pull request by babich-a**

### CODE REVIEW

File: js/ui/themes.js [hunk 1]
```diff
@@ -328,6 +329,60 @@ function checkThemeDeprecation() {
...
+    const testElement = document.createElement("span");
+
+    testElement.style.position = "absolute";
+    testElement.style.top = "-9999px";
+    testElement.style.left = "-9999px";
+    testElement.style.visibility = "hidden";
+    testElement.style.fontFamily = etalonFont;
+    testElement.style.fontSize = "250px";
+    testElement.style.fontWeight = fontWeight;
+    testElement.innerHTML = text;
```
Comment (BingoRUS) [inline js/ui/themes.js:349]: Why can’t we always use the same text?

Comment (babich-a) [reply js/ui/themes.js:349]: Google font is divided by unicode-range. 'm' can be loaded while 'м' is not

Comment (BingoRUS) [reply js/ui/themes.js:349]: That's awful 😑

Comment (babich-a) [reply js/ui/themes.js:349]: They do it now with its own guidelines :) https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/webfont-optimization?hl=en#optimization_checklist

**Review by jsdmitry — APPROVED** [review]

**Review by groshenkovamarina — APPROVED** [review]

**Pull request merged**